server/controllers/UserController.js

```diff
@@ -54,12 +56,23 @@ export default class UsersController {
   }
 
   /**
-  * Gets the current/authenticated user using the id.
+  * Gets the current/authenticated user using the username.
   * @param {object} req the request object
   * @param {object} res the response object
   * @returns {user} the user object
   */
-  static FindUserById(req, res) {
+  static getLoggedInUser(req, res) {
+    // This is just to test the isLoggedIn middleware.
+    // The real immplementation may be different.
+    const { username } = req.user;
+    User.findOne({
+      where: { username },
+      attributes: ['id', 'username', 'email', 'bio', 'image'],
+    })
+      .then(user => res.status(200).send({
+        status: 'success',
+        user,
+      }));
     // to be implemented
   }
 
```

Comment:
I noticed you changed the name of the file. Well done. However, your class name is still pluralised. Please review and make the necessary change as soon as you can.